abook.py

```diff
@@ -396,4 +411,27 @@ def vcf2abook() -> None:
     )
     args = parser.parse_args()
 
-    Abook.abook_file(args.infile, args.outfile)
+    
+    if os.path.isfile(args.outfile):
+        abook = Abook(args.outfile)
+        abook_uids = abook.get_uids()
+        full_names = []
+        # get full names of incoming vcf contact
+        for line in args.infile:
```

Comment:
I would use `readComponents` here.
Also I would prefer to merge by UID but we can discuss this.

Reply:
ok great thanks